### ISSUE TITLE
Generated AppImage has strange name

### ISSUE THREAD

**Issue by probonopd** [matthewbauer/nix-bundle#2]
The Firefox AppImage gets generated in a strange place:

```
me@host:~/nix-bundle-master$ ls -lh /nix/store/jbwdybyyfs4y4yzdivwa205s82nhbzfv-appimage.AppImage 
-r-xr-xr-x 1 me me 127M Jan  1  1970 /nix/store/jbwdybyyfs4y4yzdivwa205s82nhbzfv-appimage.AppImage
```
That is a strange name.

If appimagetool would only be invoked with `SOURCE` as an argument, then it would determine the name of the AppImage from the application name in the desktop file.

**Comment by probonopd** [comment]
Apart from that, the Firefox AppImage got built correctly. Contains, as of today, "Nightly 51.0.1 (64-bit)".

**Comment by matthewbauer** [comment]
This should be fixed with 51c628cb0202a7a299adafbe4d198ada5766833e.
